azuredevops/internal/service/serviceendpoint/resource_serviceendpoint_devops.go

```diff
@@ -0,0 +1,105 @@
+package serviceendpoint
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/microsoft/azure-devops-go-api/azuredevops/serviceendpoint"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils/converter"
+	"github.com/microsoft/terraform-provider-azuredevops/azuredevops/internal/utils/tfhelper"
+)
+
+// ResourceServiceEndpointAzureDevOps schema and implementation for Azure DevOps service endpoint resource
+func ResourceServiceEndpointAzureDevOps() *schema.Resource {
+	r := genBaseServiceEndpointResource(flattenServiceEndpointAzureDevOps, expandServiceEndpointAzureDevOps)
+	r.Schema["organization_name"] = &schema.Schema{
+		Type:        schema.TypeString,
+		Required:    true,
+		Description: "Azure DevOps organization name",
+	}
+
+	r.Schema["auth_personal"] = &schema.Schema{
+		Type:     schema.TypeSet,
+		Required: true,
+		MinItems: 1,
+		MaxItems: 1,
+		Elem:     azdoPersonalAccessTokenField(),
+	}
+
+	return r
+}
+
+// Convert internal Terraform data structure to an AzDO data structure:
+func expandServiceEndpointAzureDevOps(d *schema.ResourceData) (*serviceendpoint.ServiceEndpoint, *string, error) {
```

Comment:
function `expandServiceEndpointAzureDevOps` never throw an error, I would suggest remove the `error ` from the return parameter.

Reply:
done